target/linux/ramips/dts/GB-PC1.dts

```diff
@@ -0,0 +1,157 @@
+/dts-v1/;
+
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	model = "GB-PC1";
```

Comment:
Sure I can add one ... But is a "compatible string" really required ?
Seems to work OK without one.